README.md

```diff
@@ -7,17 +7,293 @@ SDK for the Seam API written in Ruby.
 
 ## Description
 
-TODO
+[Seam](https://seam.co) makes it easy to integrate IoT devices with your applications.
+This is an official SDK for the Seam API.
+Please refer to the official [Seam Docs](https://docs.seam.co/latest/) to get started.
+
+Parts of this SDK are generated from always up-to-date type information
+provided by [@seamapi/types](https://github.com/seamapi/types/).
+This ensures all API methods, request shapes, and response shapes are
+accurate and fully typed.
 
 ## Installation
 
-Add this as a dependency to your project using [Bundler] with
+Add this as a dependency to your project using [Bundler](https://bundler.io/) with:
 
-```
+```bash
 $ bundle add seam
 ```
 
-[bundler]: https://bundler.io/
+## Usage
+
+### Examples
+
+**Note:** _These examples assume `SEAM_API_KEY` is set in your environment._
```

Comment:
Let's upgrade this and the other README to use the GH note syntax
![Uploading image.png…]()

```suggestion
> [!NOTE]
> These examples assume `SEAM_API_KEY` is set in your environment.
```

Reply:
b902492cc500c0d0e6d79b341a81b1b439c978ec